frontend/webpack.config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     hints: false
   },
   module: {
-    rules: [isProduction ? {} : {
+    rules: [true ? {} : {
```

Comment:
always true?

Reply:
:scream: that should not have been committed. Thanks.